systemtest/src/test/java/io/strimzi/systemtest/TopicST.java

```diff
@@ -181,6 +195,45 @@ void testBigAmountOfTopicsCreatingViaKafka() {
         }
     }
 
+    @Tag(NODEPORT_SUPPORTED)
+    @Tag(SCALABILITY)
+    @Test
+    void testBigAmountOfTopicsCreatingViaAdminClient() throws ExecutionException, InterruptedException, TimeoutException {
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3, 1)
+            .editSpec()
+                .editKafka()
+                    .editListeners()
+                        .withNewKafkaListenerExternalNodePort()
+                            .withTls(false)
+                        .endKafkaListenerExternalNodePort()
+                    .endListeners()
+                .endKafka()
+            .endSpec()
+            .done();
+
+        Properties properties = new Properties();
+        properties.setProperty(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, AbstractKafkaClient.getExternalBootstrapConnect(NAMESPACE, CLUSTER_NAME));
+
+        try (AdminClient adminClient = AdminClient.create(properties)) {
+
+            for (int i = 0; i < NUMBER_OF_TOPICS; i++) {
+                String topicName = TOPIC_NAME + "-" + i;
+                LOGGER.debug("Creating async topic {} via Admin client", topicName);
+                CreateTopicsResult crt = adminClient.createTopics(singletonList(new NewTopic(topicName, 1, (short) 1)));
+                crt.all().get();
+            }
+
+            Set<String> topics = adminClient.listTopics().names().get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS);
+
+            LOGGER.info("Verify that in Kafka cluster contains {} topics", NUMBER_OF_TOPICS);
+            assertThat(topics.size(), is(NUMBER_OF_TOPICS));
+        }
+
+        LOGGER.info("Verify that {} topics are inside Kubernetes CR", NUMBER_OF_TOPICS);
```

Comment:
I would say "Verify that corresponding {} KafkaTopic custom resources were created"